src/nupic/experimental/ExtendedTemporalMemory.cpp

```diff
@@ -1300,6 +1331,11 @@ void ExtendedTemporalMemory::write(ExtendedTemporalMemoryProto::Builder& proto)
     matchingApicalSegmentOverlaps[i].setOverlap(
       matchingApicalSegments_[i].overlap);
   }
+
+  NTA_CHECK(learnOnOneCell_ == false) <<
+    "Serialization is not supported for learnOnOneCell";
```

Comment:
And the `save` doesn't appear to have the same check. But I'd rather support both.